### PR TITLE
Persistent mounting integration tests

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description

Test all the integration tests with [persistent mounting](https://cloud.google.com/storage/docs/gcsfuse-mount).
Before we were only setting bin file. created getter-setter for bin and sbin file. In mounting function added argument to pass bin or sbin file.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Run all the tests for testbucket and mountedDirectory flags.(status:SUCCESS)
11. Unit tests - NA
12. Integration tests - Run all the tests for testbucket and mountedDirectory flags.(status:SUCCESS)

### CODE REVIEW

File: tools/integration_tests/util/mounting/persistent_mounting/perisistent_mounting.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		s = strings.Replace(s, "-", "_", -1)
+		// e.g. Convert __enable_storage_client_library to enable_storage_client_library
+		s = strings.Replace(s, "__", "", -1)
+		args = append(args, s)
+	}
+	return
+}
+
+func mountGcsfuseWithStaticMounting(flags []string) (err error) {
+	defaultArg := []string{setup.TestBucket(),
```
Comment (raj-prince) [inline tools/integration_tests/util/mounting/persistent_mounting/perisistent_mounting.go:43]: What happens if we pass some argument repeated while mounting the gcsfuse?

Are we sure, the input argument flags doesn't contain defaultArg?

File: tools/integration_tests/util/mounting/persistent_mounting/perisistent_mounting.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	for i := 0; i < len(flags); i++ {
+		if err = mountGcsfuseWithStaticMounting(flags[i]); err != nil {
+			setup.LogAndExit(fmt.Sprintf("mountGcsfuse: %v\n", err))
+		}
+		setup.ExecuteTestForFlagsSet(flags[i], m)
+	}
+	return
+}
+
+func RunTests(flags [][]string, m *testing.M) (successCode int) {
```
Comment (raj-prince) [inline tools/integration_tests/util/mounting/persistent_mounting/perisistent_mounting.go:84]: What happens if we provide persistent-mounting type flags?

**Review by raj-prince — COMMENTED** [review]
Added few comments.

Overall - how much time execution of integration test take now?

File: tools/integration_tests/util/mounting/persistent_mounting/perisistent_mounting.go (hunk 2 above)
Comment (Tulsishah) [reply tools/integration_tests/util/mounting/persistent_mounting/perisistent_mounting.go:84]: It will convert it in persistent_mounting

**Comment by Tulsishah** [comment]
> Added few comments.
> 
> Overall - how much time execution of integration test take now?

Between 25 to 30 minutes

**Review by raj-prince — APPROVED** [review]

**Pull request merged**